src/receta/Condimento.java

```diff
@@ -15,4 +15,11 @@ public Condimento(String tipoCondimento){
         super();
         this.setNombre(tipoCondimento);
     }
+    
+    @Override
+    public String toString() {
+        return "\n     *Nombre: " + this.getNombre()+

```

Comment:
Respecto a la operacion 
       if(this.getCantidad() > cantidad){ 
            if(this.getMedida().equals(medida)){
                this.setCantidad(this.getCantidad()- cantidad);

lo que entiendo es que si hay ingredientes en la cocina, pues puedo usar ingredientes en la receta. Ahora lo que me confunde un poco es que se haria el llamado en el main, porque como le digo que this.getCantidad() es el del inventario si el que me entra es de la receta > cantidad. No se si entendi mal, alli me siento un poco confundida, entiendo la logica y lo que hace, lo que no entiendo es como podriamos lograr hacer la operacion. Me hago entender?